modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path

```diff
@@ -133,6 +133,21 @@
 	<td>//tr[contains(.,'${key_workflowTask}') and contains(.,'${key_workflowAssetTitle}')]/td[6]//div/a</td>
 	<td></td>
 </tr>
+<tr>
+	<td>MY_WORKFLOW_TASKS_SEARCH_BAR</td>
+	<td>//input[@placeholder='Search for']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MY_WORKFLOW_TASKS_SEARCH_BUTTON</td>
+	<td>//button[contains(@type,'submit')]//*[contains(@class,'lexicon-icon-search')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MY_WORKFLOW_TASKS_SEARCH_RESULTS_BAR</td>
+	<td>//span[contains(.,'Results') and @class='text-truncate']</td>
+	<td></td>
+</tr>
```

Comment:
You can sync those paths created with other created by Paulo Alves at that [PR](https://github.com/rodrigocunhaa/liferay-portal/pull/823/files). 
 
Suggestion:

MY_WORKFLOW_TASKS_SEARCH_INPUT_FIELD
MY_WORKFLOW_TASKS_SEARCH_ICON
MY_WORKFLOW_TASKS_SEARCH_RESULT_BAR